packages/uswds-core/src/styles/_notifications.scss

```
@@ -206,6 +206,15 @@ $uswds-notifications:
 + "\A   from announcing the invalid file type error message."
 + "\A - Accessibility: [usa-footer] Removed `overflow: hidden` from `usa-footer` "
 + "\A   to allow the full focus outline to show."
++ "\A "
++ "\A --------------------------------------------------------------------"
++ "\A 3.12.0"
++ "\A Release notes: github.com/uswds/uswds/releases/tag/v3.12.0"
++ "\A "
++ "\A - Display: [usa-in-page-navigation]"
++ "\A   Fixed a bug that prevented in-page navigation from scrolling to nested headings."
++ "\A   ✏️ Teams that use `data-scroll-offset` should confirm this change"
++ "\A     does not cause regressions in scroll behaviors."
```

**issue (non-blocking)**: @amyleadem the other in-page navigation change also has an edit note and could be useful.

```suggestion
+ "\A     does not cause regressions in scroll behaviors."
+ "\A - Display: [usa-in-page-navigation]"
+ "\A   Added the `data-minimum-heading-count` property to the in-page navigation component."
+ "\A   ✏️ Teams should customize the value of this property based on their content needs."
```

---

I was torn on including this one because I didn't see other references to `data-` properties in the file. I support including it because teams might need to take action on it, but going to hold off on committing this in case anyone else has thoughts.

---

I think my perspective here when it comes to notifications, is that we focus on what might potentially be breaking or what needs manual intervention. In this case, it's not that devs should use this new feature, but that they should confirm that it does not cause regressions on pages with only a single header

---

```suggestion
+ "\A     does not cause regressions in scroll behaviors."
+ "\A - Display: [usa-in-page-navigation]"
+ "\A   Added the `data-minimum-heading-count` property to the in-page navigation component."
+ "\A   ✏️ Teams should check to make sure the default minimum of 2 does not result"
+ "\A   in regressions."
Commit suggestion
Add suggestion to batch
```

Something like that?